types/src/transaction/transaction.rs

```diff
@@ -461,6 +461,46 @@ impl PartialHashing for Transaction {
                     &blake128(tag.get_tag()),
                 ))
             }
+            Transaction::AssetCompose {
+                network_id,
+                shard_id,
+                world_id,
+                nonce,
+                metadata,
+                registrar,
+                inputs,
+                output,
+            } => {
+                if tag.filter_len != 0 {
```

Comment:
Why must `filter_len` be zero?

Reply:
There is only 1 output for `AssetCompose`. `output = all` and `output = none` can describe all the case.